pandas-stubs/core/window/__init__.pyi

```diff
@@ -8,3 +11,13 @@ from pandas.core.window.rolling import (
     RollingGroupby as RollingGroupby,
     Window as Window,
 )
+
+__all__ = [
+    "Expanding",
+    "ExpandingGroupby",
+    "ExponentialMovingWindow",
+    "ExponentialMovingWindowGroupby",
+    "Rolling",
+    "RollingGroupby",
+    "Window",
+]
```

Comment:
Since none of these should be instantiated by a user, do we still need to have `__all__` here even though they are returned by various pandas methods?

Reply:
Will remove